docs/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
```

Comment:
Add `set -e`, if some command fails the script should stop and report error

Reply:
OK